content/doc/developer/publishing/releasing-experimental-updates.adoc

```diff
@@ -6,6 +6,8 @@ layout: developer
 To simplify delivery of beta versions of plugins to interested users, the Jenkins project publishes an _experimental update center_.
 It includes alpha and beta versions of plugins, which aren't usually included in the regular update center.
 
+navigate to the "Manage Jenkins" menu and select "Plugin Manager.",Select the "Advanced" tab. Under the "Upload Plugin" section, select the "Choose File" button and select the unreleased plugin version you want to test.Select the "Upload" button to upload the plugin.In the "Plugin Manager" page, select the "Incremental Build" option.Select the "Test" button to test the plugin. Once the plugin has been tested and is deemed stable, you can then install it on your production environment.It is also good practice to make sure to backup your Jenkins configuration and the plugins before installing new plugins and testing it in a test environment before deploying it in production.
+
```

Comment:
The notes in the [PDF file](https://github.com/jenkins-infra/jenkins.io/files/10476776/Publishing.Experimental.Plugin.Releases.pdf) said to describe the use of the plugin installation manager tool.  This paragraph describes how to install an unreleased plugin without using the experimental update center.  I don't think that we want this paragraph here, because this page is focused on the experimental update center, not on the broader process of testing unreleased plugins.

Darin Pope has created a [plugin installation manager tool video](https://www.youtube.com/watch?v=vrttfNgB1hk) that shows how to use it.  There may be a description and phrasing in that video that will help you create the text that needs to be inserted in this paragraph.

An edited version of this paragraph might fit elsewhere in the documentation, but not in the page on experimental plugin releases.
```suggestion
```